.mention-bot

```diff
@@ -7,6 +7,6 @@
     "Misc/NEWS"
   ],
   "userBlacklist": [],
-  "userBlacklistForPR": [],
+  "userBlacklistForPR": ["benjaminp"],
```

Comment:
If you intended to make it so mention-bot never @ mentions you personally, then you need to add yourself to ``userBlacklist`` not ``userBlacklistForPR``. The later will make it so mention-bot never mentions *anyone* on PRs created by you.